config.json

```diff
@@ -14,6 +14,18 @@
         "initiateLifetime": {
             "hours": 1
         },
+        "loginDir": "/login/",
+        "loginHash": "sha512",
+        "loginIdLength": 24,
+        "loginLifetime": {
+            "minutes": 15
+        },
+        "passwordHash": {
```

Comment:
HOWEVER, we don't want "pbkdf2" because that's a specific implementation.  How about "secureHash"?  And we should make "pbkdf2" a value...

```
secureHash: {
    algo: sha512,
    derivation: pbkdf2,
    bytes: 48,
    iterations: 100000,
    saltLength: 64
}